suites/pacific/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml

```diff
@@ -0,0 +1,205 @@
+              command: apply
+              service: prometheus
+              base_cmd_args:
+                verbose: true
+              args:
+                placement:
+                  nodes:
+                    - node1
+                  limit: 1
+                  sep: ";"
+          - config:
+              command: apply
+              service: grafana
+              base_cmd_args:
+                verbose: true
+              args:
+                placement:
+                  nodes:
+                    - node1
+                  limit: 1
+                  sep: ";"
+          - config:
+              command: apply
+              service: alertmanager
+              args:
+                placement:
+                  label: alertmanager
+          - config:
+              command: apply
+              service: node-exporter
+              base_cmd_args:
+                verbose: true
+              args:
+                placement:
+                  nodes: "*"
+          - config:
+              command: apply
+              service: crash
+              base_cmd_args:
+                verbose: true
+              args:
+                placement:
+                  nodes: "*"
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Configure client
+      desc: Configure client with admin keyring.
+      module: test_client.py
+      polarion-id:
+      config:
+        command: add
+        id: client.1                      # client Id (<type>.<Id>) --> /etc/ceph/ceph.{id}.keyring
+        node: node4                       # client node
+        copy_admin_keyring: true          # Copy admin keyring to node
+        store-keyring: /etc/ceph/ceph.client.1.keyring  # local copy at the installer
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Upgrade along with IOs
+      module: test_parallel.py
+      parallel:
+        - test:
+            name: rbd-io
+            module: rbd_faster_exports.py
+            config:
+              io-total: 100M
+            desc: Perform export during read/write,resizing,flattening,lock operations
+        - test:
+            name: Upgrade ceph
+            desc: Upgrade cluster to latest version
+            module: test_cephadm_upgrade.py
+            polarion-id: CEPH-83573791,CEPH-83573790
+            config:
+              command: start
+              service: upgrade
+              base_cmd_args:
+                verbose: true
+              benchmark:
+                type: rados
+                pool_per_client: true
+                pg_num: 128
+                duration: 10
+              verify_cluster_health: true
+            destroy-cluster: false
+      desc: Running upgrade and i/o's parallelly
+      abort-on-fail: true
+
+  - test:
```

Comment:
The test requires a ceph cluster with a lower version of ceph 5 and no rgw pool existing. Adding the suggested changes to a new suite without rgw configured. A corresponding conf is also created without the rgw details.